Source/monstdat.h

```diff
@@ -222,9 +209,6 @@ enum _monster_id : int16_t {
 	MT_BTBLACK,
 	MT_RBLACK,
 	MT_UNRAV,
-	MT_HOLOWONE,
-	MT_PAINMSTR,
-	MT_REALWEAV,

```

Comment:
Deleting thease will shift the ids which is probably going to break a ton of things?

Reply:
I tested it out, no crashes or issues as far as I can tell.

Reply:
Did you load a single player savegame?

Reply:
Just teleported through all crypted, killed na krul. all monsters maintained the correct data and names

Reply:
I would also expect this to break loading some quest levels.

Reply:
> Did you load a single player savegame?

no, let me test

Reply:
Just tested, no issues

Reply:
> I would also expect this to break loading some quest levels.

I will test

Reply:
No crash in Leoric's Tomb. Should I test more levels?

Reply:
aren't most of these monsters fully functional? Removing working stuff like this isn't beneficial at all